src/transaction/log_tran_table.c

```diff
@@ -4099,6 +4134,13 @@ logtb_get_mvcc_snapshot (THREAD_ENTRY * thread_p)
  */
 void
 logtb_complete_mvcc (THREAD_ENTRY * thread_p, LOG_TDES * tdes, bool committed)
+{
+  constexpr bool check_null_last_mvcc_lsa = true;
```

Comment:
Just out of curiosity, why don't you pass true directly, but use a temporal constexpr variable? Thanks.

Reply:
No other reason than to make the semantics of the code clearer to the reader. With only a `true` passed as argument, the user would have to search the called function to see what that means. It is zero-cost from a performance point of view as a compiler, in release build, will abstract away the parameter altogether.